### PR TITLE
appr: add Stewart

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

**Comment by garyverhaegen-da** [comment]
cc @stewartstewart-da

File: release/rotation [hunk 1]
```diff
@@ -2,6 +2,7 @@
 # to access an ad-hoc machine for Windows testing. This can be done by opening a PR
 # like the following: https://github.com/DACH-NY/daml-language-ad-hoc/pull/37
 U0PNNPY05 robin-da
+U03518HUVBP stewartstewart-da
```
Comment (garyverhaegen-da) [inline release/rotation:5]: we do think it's good to do that fairly quickly after joining, but I figured _tomorrow_ was perhaps a bit too harsh with the 2.0 thing going, so left @robin-da in charge of that.

**Review by stewSquared — APPROVED** [review]

File: release/rotation [hunk 2]
```diff
@@ -2,6 +2,7 @@
 # to access an ad-hoc machine for Windows testing. This can be done by opening a PR
 # like the following: https://github.com/DACH-NY/daml-language-ad-hoc/pull/37
 U0PNNPY05 robin-da
+U03518HUVBP stewSquared
```
Comment (stewSquared) [inline release/rotation:5]: confirmed :+1:

**Review by stewartstewart-da — APPROVED** [review]

File: release/rotation (hunk 2 above)
Comment (garyverhaegen-da) [reply release/rotation:5]: This means you'll be in charge of the 2.1 snapshot release on Wed 2022-03-09.

Comment (stewSquared) [reply release/rotation:5]: Noted! Happy to learn how it works

**Pull request merged**